packages/discord.js/typings/index.test-d.ts

```diff
@@ -197,6 +197,10 @@ const client: Client = new Client({
   }),
 });
 
+if (client.isReady()) {
```

Comment:
Please also add one for the opposite case, since that was actually the problematic one.